pymc3/distributions/distribution.py

```diff
@@ -339,7 +341,7 @@ def dist(
 
 @singledispatch
 def _get_moment(op, rv, size, *rv_inputs) -> TensorVariable:
-    return None
+    raise NotImplementedError(f"For random variable {rv} of type {op} has no get_moment.")
```

Comment:
```suggestion
    raise NotImplementedError(f"random variable {rv} of type {op} has no get_moment.")
```

Reply:
applied in https://github.com/pymc-devs/pymc3/pull/5025